src/v3/src/util/passwordUtils.ts

```diff
@@ -172,7 +172,7 @@ export const validatePassword = (
 ): PasswordValidation => {
   let passwordValidations: PasswordValidation = {};
 
-  if (!settings || (!settings.complexity && !settings.age)) {
+  if (!settings?.complexity || !Object.keys(settings.complexity).length) {
```

Comment:
nit: let's check length  === 0 or something rather than use coercion of `0` to `falsy`